apps/entity-service/src/entities/processors/nursery.processor.ts

```diff
@@ -140,4 +149,17 @@ export class NurseryProcessor extends EntityProcessor<Nursery, NurseryLightDto,
       )[0].seedlingsYoungTrees ?? 0
     );
   }
+
+  async delete(nursery: Nursery, userId: number) {
+    const user = await User.findOne({ where: { id: userId }, include: [Role] });
```

Comment:
Processors can now get the user id through their entity service reference:
```
const user = await User.findOne({ where: { id: this.entitiesService.userId }, include: [Role] });
```

However, I don't think you should be fetching the user record here. See below.

Reply:
removed